alacritty_terminal/src/event_loop.rs

```diff
@@ -374,17 +261,15 @@ where
                             }
                         },
 
-                        token
-                            if token == self.pty.read_token()
-                                || token == self.pty.write_token() =>
-                        {
-                            #[cfg(unix)]
+                        token if token == tty::PTY_READ_TOKEN || token == tty::PTY_WRITE_TOKEN => {
+                            /* FIXME: once polling supports extra reported modes bring it back.
```

Comment:
In general, I don't think it provides that much value given that it worked only on Linux before.

But see https://github.com/smol-rs/polling/issues/112 , I remember that it'll require a breaking bump of polling.

Reply:
I'm pretty sure without this it might cause occasional errors on exit?

Reply:
It could cause an error when you try to read again, but we have those errors on anything other than linux already (if we even have them). You may even notice that this change was added as enhancement in the past.

Reply:
As @kchibisov commented, this is just an optimization for now. It can be added back once we add greater event granularity to `polling`.

Reply:
Either way I don't like the documentation in the comments. If this is something we want to track, it should be removed from the code and added to the issue tracker.